stable/grafana/values.yaml

```diff
@@ -214,3 +215,23 @@ smtp:
   # `existingSecret` is a reference to an existing secret containing the smtp configuration
   # for Grafana in keys `user` and `password`.
   existingSecret: ""
+
+## Sidecars that collect the configmaps with specified label and stores the included files them into the respective folders
+## Requires at least Grafana 5 to work and can't be used together with parameters dashboardProviders, datasources and dashboards
+sidecar:
+  image: kiwigrid/k8s-sidecar:0.0.3
+  imagePullPolicy: Always
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 100Mi
+    requests:
```

Comment:
Don't specify defaults for resources. Commeent them out.

Reply:
May i ask why?